app/src/main/java/com/odoo/table/ResPartner.java

```diff
@@ -32,6 +32,6 @@ public ResPartner(Context context) {
     }
 
     public Uri uri() {
-        return ContactDatabaseProvider.buildURI(AUTHORITY, getTableName());
+        return ContactProvider.buildURI(AUTHORITY, getTableName());
     }
```

Comment:
does not required to build uri for model. We use ResPartner directly in ContentProvider.